opendb-core/src/main/java/org/openplacereviews/opendb/api/ApiController.java

```diff
@@ -109,6 +93,12 @@ public String queueList() throws FailedVerificationException {
 		return formatter.fullObjectToJson(bl);
 	}
 
+	@GetMapping(path = "/queue/size")
```

Comment:
Is this method needed? We have blockchain status class, so we don't need separate query